dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -144,7 +144,7 @@ export type Props = {
 	 * For example, the first splash card in the second collection would be: "collection-1-splash-0"
 	 */
 	uniqueId?: string;
-	/** The Splash card in a flexible container gets a different visual treatment to other cards */
+	/** The Splash card in a flexible or onward container gets a different visual treatment to other cards */
 	isFlexSplash?: boolean;
```

Comment:
I wonder if we should rename this field. 

Previously, its been used to refer to a splash card in a flexible container. Perhaps this is too specific now if the affects are always shared with an onward container. 

Perhaps just splash card is ok. Does the onwards "splash" card share the same characteristics as a flexible splash card?